### PR TITLE
feat(harpoon): Update to v2

### PR DESCRIPTION
**Pull request by luxus**
tested with minimal v3 install.. but not actually using it. so i hope it works :D

### CODE REVIEW

**Comment by luxus** [comment]
it should work now

**Comment by Uzaaft** [comment]
@luxus  I’ll take a look in about 1h. 👀

File: lua/astrocommunity/motion/harpoon/init.lua [hunk 1]
```diff
@@ -7,38 +7,44 @@ require("astronvim.utils").set_mappings(maps)
 return {
   {
     "ThePrimeagen/harpoon",
+    branch = "harpoon2",
     dependencies = {
       "nvim-lua/plenary.nvim",
       "nvim-telescope/telescope.nvim",
     },
     cmd = { "Harpoon" },
```
Comment (Uzaaft) [inline lua/astrocommunity/motion/harpoon/init.lua:15]:
```suggestion
    cmd = { "Harpoon" },
```
The command Harpoon isn't exposed by the plugin

Comment (Uzaaft) [inline lua/astrocommunity/motion/harpoon/init.lua:15]:
```suggestion
```
The command Harpoon isn't exposed by the plugin

**Review by Uzaaft — CHANGES_REQUESTED** [review]

**Review by Uzaaft — APPROVED** [review]

**Comment by mehalter** [comment]
This was incorrectly not marked as a breaking change

**Comment by mehalter** [comment]
just a heads up. I will revise the commit now before we tag a release

**Pull request merged**